### ISSUE TITLE
Show a DisplayAlert over the popup

### ISSUE THREAD

**Issue by acaliaro** [MPowerKit/Popups#5]
Hi @Alex-Dobrynin thanks for your work.
I have a question.
If I use this code

```
private  async void OnCounterClicked(object sender, EventArgs e)
    {
        count++;

        if (count == 1)
            CounterBtn.Text = $"Clicked {count} time";
        else
            CounterBtn.Text = $"Clicked {count} times";

        await PopupService.ShowPopupAsync(new PopupTestPage());
        await Task.Delay(2000);
        App.Current.MainPage.DisplayAlert("AA", "bb", "ok");
        await Task.Delay(2000);

        await PopupService.HidePopupAsync();

    }


```
I have a displayalert that is "under" the loader. Is it possible to have it over the loader so I can interact with it?

the problem is only on iOS. on Android it seems to work fine (displayalert is over the loader so I can interact with it)

**Comment by acaliaro** [comment]
maybe it is related to https://github.com/LuckyDucko/Mopups/issues/66 ?

**Comment by Alex-Dobrynin** [comment]
Yes this is the same issue. and there you can find workarounds and why it works this way.

but I have a question: why you are using this package instead of mopups? (just collection some analytics).

I can suggest you to use Controls.UserDialogs.Maui instead of `App.Current.MainPage.DisplayAlert("AA", "bb", "ok");`, but im not sure if there is the same issue
